### PR TITLE
Fix extendability access points for overwriting pieces of the importer

### PR DESCRIPTION
**Pull request by emdashcodes**
## What/Why

When we changed a few functions to return information instead of echoing it to the screen (https://github.com/WordPress/tumblr-importer/pull/14), it broke compatibility with WordPress.com's overwriting of certain pieces.

## Testing

* Run a smoke test import locally, I'll be updating the WordPress.com patch shortly

### CODE REVIEW

File: src/class-tumblr-import.php [hunk 1]
```diff
@@ -311,7 +311,7 @@ public function show_blogs( $error = null ) {
 		}
 
 		$output .= '<h2>' . esc_html__( 'Import Tumblr', 'tumblr-importer' ) . '</h2>';
-		$output .= do_action( 'tumblr_importer_import_instructions' );
```
Comment (emdashcodes) [inline src/class-tumblr-import.php:314]: do_action does not return anything, so this never gets set.

File: src/class-wp-importer-cron.php [hunk 2]
```diff
@@ -11,6 +11,10 @@
 
 defined( 'ABSPATH' ) || exit;
 
+if ( class_exists( 'WP_Importer_Cron' ) ) :
```
Comment (emdashcodes) [inline src/class-wp-importer-cron.php]: A check for this was removed when moving things to the `src/` directory.

WordPress.com implements it's own `WP_Importer_Cron`.

Comment (rtio) [reply src/class-wp-importer-cron.php]: is there a reason for not using `{}`?

**Review by rtio — APPROVED** [review]
just tested, this is working fine

**Pull request merged**